content/en/platform/corda/5.0-dev-preview-2/getting-started/cordapp-standard-development-environment/csde.md

```diff
@@ -9,25 +9,35 @@ menu:
 section_menu: corda-5-dev-preview2
 ---
 The CorDapp Standard Development Environment (CSDE) makes the process of prototyping CorDapps on Developer Preview 2 more straight-forward.
-The CSDE is obtained by cloning our `CSDE-Cordapp-Template-Kotlin` repository to your local machine. The CSDE provides:
-* A prepared CorDapp project that you can use as a starting point to develop your own prototypes
-* A base Gradle configuration that brings in the dependencies you need to write and test a Corda 5 CorDapp
-* A set of Gradle helper tasks which speed up and simplify the development and deployment process; these are effectively wrappers over the [Corda CLI](../../developing/corda-cli/overview.html)
-* Debug configuration for debugging a local Corda cluster
-* The `MyFirstFlow` code which forms the basis of the Getting Started documentation
-* The ability to configure the members of the local Corda network
+The CSDE is obtained by cloning our `CSDE-Cordapp-Template-Kotlin` or `CSDE-Cordapp-Template-java` repository to your local machine. The CSDE provides:
```

Comment:
Only CSDE is in capitals for the 2 repo names.
So: CSDE-Cordapp-Template-kotlin and CSDE-cordapp-template-java